verification/curator-service/ui/README.md

```diff
@@ -0,0 +1,42 @@
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
should we use a default port then now that it's a different app?
3002 would be next

Reply:
Good catch, updated.